### PR TITLE
added Twilio Lookup API

### PR DESCRIPTION
**Pull request by namanjoshi19**
Returns phone number information matching the specified request. Formatting information is standard. Carrier, Caller Name, and phone number type information can be requested, in addition to using Add-ons to access 3rd party data sources.

### CODE REVIEW

File: src/content/apis/SMS/TWILIO LOOKUP [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Twilio Lookup API
+description: Returns phone number information matching the specified request.
+categories:
+ -
```
Comment (TheLearneer) [inline src/content/apis/SMS/TWILIO LOOKUP:5]: Please add the appropriate category. For this case it should be `development`

File: src/content/apis/SMS/TWILIO LOOKUP [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Twilio Lookup API
+description: Returns phone number information matching the specified request.
+categories:
+ - 
+features:
+  https: true
+  pricing: free
```
Comment (TheLearneer) [inline src/content/apis/SMS/TWILIO LOOKUP:8]:
```suggestion
  pricing: upgradable
```

File: src/content/apis/SMS/TWILIO LOOKUP [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+url: https://rapidapi.com/twilio/api/twilio-lookup/
```
Comment (TheLearneer) [inline src/content/apis/SMS/TWILIO LOOKUP:9]: Twilio has their own documentation page, please add the link to it 🙏

File: src/content/apis/SMS/TWILIO LOOKUP [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+logo:
```
Comment (TheLearneer) [inline src/content/apis/SMS/TWILIO LOOKUP:10]: Please add the logo of Twilio 🙏

**Review by TheLearneer — CHANGES_REQUESTED** [review]

**Review by TheLearneer — CHANGES_REQUESTED** [review]
@namanjoshi19 The file format is completely incorrect, please check [CONTRIBUTING.md](https://github.com/TheLearneer/api-party/blob/main/CONTRIBUTING.md) and make the changes accordingly

**Comment by namanjoshi19** [comment]
can i add the logo directly to the file ?

**Review by TheLearneer — COMMENTED** [review]
The file doesn't follow the contributing guidelines 🙏

**Pull request closed without merging**